content/cockpit-134.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 134
+author: Dominik Perpeet
+date: 2017-03-09 18:00
+tags: cockpit, linux
+slug: cockpit-134
+category: release
+summary: Cockpit is now fully translatable
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 134.
+
+### Login page is now translatable
+
+The majority of Cockpit has long been translatable. Since the login page is a bit of a special case, it required some
+special attention to make that also play well with the language display settings. Take a look at the video below for a demo.
+Check out the Zanata link below, it's very easy to contribute translations to Cockpit.
+At this time our top 3 translations (>90%) are Polish (pl), Ukranian (uk) and Chinese (zh-CN). Every bit of help
```

Comment:
I appreciate the irony in making typos while nitpicking about other people's typos :)